AcsfToolsCommands.php

```diff
@@ -148,6 +150,9 @@ public function sitesInfo() {
    *   Run cache clear on all sites with delay of 10 seconds between each site.
    * @usage drush acsf-tools-ml cron --use-https=1
    *   Run cron on all sites using secure url for URI.
+   * @usage drush acsf-tools-ml cron --use-https=1 --domain-containing=collection
```

Comment:
Please remove --use-https=1 from this example which is here to demo the pattern feature instead.